bigquery/google/cloud/bigquery/table.py

```diff
@@ -1849,7 +1849,7 @@ def from_api_repr(cls, api_repr):
             google.cloud.bigquery.table.TimePartitioning:
                 The ``TimePartitioning`` object.
         """
-        instance = cls(api_repr["type"])
+        instance = cls()
```

Comment:
Shouldn't this preserve the `type` from `api_repr`, if present?  Otherwise, the ctor is going to overwrite it with `TimePartitioningType.DAY`.

```suggestion
        instance = cls(api_repr.get("type"))
```

Reply:
I originally had what you suggest, but then I realized whatever we pass in will get overwritten by the next line which overwrites the whole `_properties` dictionary.

Reply:
I missed that.